doc/new-tracing/tracers_doc_generated.md

```diff
@@ -1,556 +1,561 @@
-2.34 [cardano.node.mempoolBytes] (#cardanonodemempoolbytes)
-2.35 [cardano.node.mempoolBytes] (#cardanonodemempoolbytes)
-2.36 [cardano.node.mempoolBytes] (#cardanonodemempoolbytes)
-2.37 [cardano.node.mempoolBytes] (#cardanonodemempoolbytes)
-2.38 [cardano.node.nodeCannotForge] (#cardanonodenodecannotforge)
-2.39 [cardano.node.nodeIsLeader] (#cardanonodenodeisleader)
-2.40 [cardano.node.nodeNotLeader] (#cardanonodenodenotleader)
-2.41 [cardano.node.notAdoptedSlotLast] (#cardanonodenotadoptedslotlast)
-2.42 [cardano.node.operationalCertificateExpiryKESPeriod] (#cardanonodeoperationalcertificateexpirykesperiod)
-2.43 [cardano.node.operationalCertificateStartKESPeriod] (#cardanonodeoperationalcertificatestartkesperiod)
-2.44 [cardano.node.peerSelection.cold] (#cardanonodepeerselectioncold)
-2.45 [cardano.node.peerSelection.hot] (#cardanonodepeerselectionhot)
-2.46 [cardano.node.peerSelection.warm] (#cardanonodepeerselectionwarm)
-2.47 [cardano.node.remainingKESPeriods] (#cardanonoderemainingkesperiods)
-2.48 [cardano.node.served.block] (#cardanonodeservedblock)
-2.49 [cardano.node.slotInEpoch] (#cardanonodeslotinepoch)
-2.50 [cardano.node.slotInEpoch] (#cardanonodeslotinepoch)
-2.51 [cardano.node.slotIsImmutable] (#cardanonodeslotisimmutable)
-2.52 [cardano.node.slots] (#cardanonodeslots)
-2.53 [cardano.node.slots] (#cardanonodeslots)
-2.54 [cardano.node.submissions.accepted] (#cardanonodesubmissionsaccepted)
-2.55 [cardano.node.submissions.rejected] (#cardanonodesubmissionsrejected)
-2.56 [cardano.node.submissions.submitted] (#cardanonodesubmissionssubmitted)
-2.57 [cardano.node.txsInMempool] (#cardanonodetxsinmempool)
-2.58 [cardano.node.txsInMempool] (#cardanonodetxsinmempool)
-2.59 [cardano.node.txsInMempool] (#cardanonodetxsinmempool)
-2.60 [cardano.node.txsInMempool] (#cardanonodetxsinmempool)
-2.61 [cardano.node.txsProcessedNum] (#cardanonodetxsprocessednum)
-2.62 [cardano.node.utxoSize] (#cardanonodeutxosize)
-2.63 [mem.resident] (#memresident)
-2.64 [peersFromNodeKernel] (#peersfromnodekernel)
-2.65 [rts.gcLiveBytes] (#rtsgclivebytes)
-2.66 [rts.gcMajorNum] (#rtsgcmajornum)
-2.67 [rts.gcMinorNum] (#rtsgcminornum)
-2.68 [rts.gcticks] (#rtsgcticks)
-2.69 [rts.mutticks] (#rtsmutticks)
-2.70 [rts.threads] (#rtsthreads)
-2.71 [stat.cputicks] (#statcputicks)
-3.[Datapoints] (#datapoints)
-1.1 [NodeInfo] (#nodeinfo)
+# Cardano Trace Documentation
+# Table Of Contents
+
+
+## [Trace Messages](#trace-messages)
+1. [Cardano.Node.AcceptPolicy.ConectionRateLimiting](#cardanonodeacceptpolicyconectionratelimiting)
+1. [Cardano.Node.AcceptPolicy.ConnectionHardLimit](#cardanonodeacceptpolicyconnectionhardlimit)
+1. [Cardano.Node.BlockFetch.NodeToNode.Send.BatchDone](#cardanonodeblockfetchnodetonodesendbatchdone)
+1. [Cardano.Node.BlockFetch.NodeToNode.Send.Block](#cardanonodeblockfetchnodetonodesendblock)
+1. [Cardano.Node.BlockFetch.NodeToNode.Send.ClientDone](#cardanonodeblockfetchnodetonodesendclientdone)
```

Comment:
Is this renumbering it all ones intentional?

Reply:
@newhoggy, we're relying on Github's markdown auto-numbering.

Reply:
oh, but we read markdowns not only on github.